source/tests/system/nixnet_utilities.h

```diff
@@ -0,0 +1,195 @@
+#include <gtest/gtest.h>
+#undef interface
+#include <nixnet/nixnet_client.h>
+
+#include <string>
+#include <vector>
+
+#define EXPECT_SUCCESS(response_)    \
+  ([](auto& response) {              \
+    EXPECT_EQ(0, response.status()); \
+    return response;                 \
+  })(response_)
+
+#define EXPECT_XNET_ERROR(error, response) \
+  if (1) {                                 \
+    EXPECT_EQ(error, (response).status()); \
+  }
+
+namespace nixnet_utilities {
+namespace {
+
+namespace client = nixnet_grpc::experimental::client;
+namespace pb = google::protobuf;
+using namespace nixnet_grpc;
+
+typedef pb::uint32 u32;
+
+GetPropertyResponse get_property(const client::StubPtr& stub, const nidevice_grpc::Session& session_ref, const client::simple_variant<nixnet_grpc::Property, pb::uint32>& property_id)
```

Comment:
Are these actually called?

Reply:
Right now, only few functions from them are being used but these all were added because they were being used in some other tests which will be added soon. If ill find any function not being used then ill remove them in subsequent PRs